parsers/cxx/cxx_parser_internal.h

```diff
@@ -220,7 +220,9 @@ typedef enum _CXXParserKeywordState
 	// "const" has been seen at block level
 	CXXParserKeywordStateSeenConst = (1 << 9),
 	// "volatile" has been seen at block level
-	CXXParserKeywordStateSeenVolatile = (1 << 10)
+	CXXParserKeywordStateSeenVolatile = (1 << 10),
+	// __attribute__((deprecated)) has been seen
+	CXXParserKeywordStateSeenAttributeDeprecated = (1 << 11)
```

Comment:
I recommend put `,` at the end of newly introduced entry. So the next time introducing new one, you don't have to append `,` at the end of its previously line. That makes changes smaller. Smaller changes help us to understand them.